src/js/samples/components/Sidebar/Selector.js

```diff
@@ -77,12 +86,12 @@ export const SampleSidebarSelector = ({
                             autoFocus
                         />
                     </SampleSidebarSelectorInputContainer>
-                    <SampleItemComponentsContainer>
-                        {sampleItemComponents}
-                        <SampleSidebarSelectorButton>
+                    <SampleItemComponentsContainer>{sampleItemComponents}</SampleItemComponentsContainer>
+                    <SampleSidebarSelectorButton>
+                        <SampleLinkContainer>
```

Comment:
You might be able to merge `<SampleSidebarSelectorButton>` and `<SampleLinkContainer>` into a single styled component. (I don't think `<SampleSidebarSelectorButton>` is used anywhere else in the component) Otherwise looks good to me!